tests/e2e/utils/RequestHandler.ts

```diff
@@ -0,0 +1,61 @@
+import { TestConstants, RequestType } from '..';
+import axios from 'axios';
+import querystring from 'querystring';
+import { injectable } from 'inversify';
+
+@injectable()
+export class RequestHandler {
```

Comment:
- Too general class
- Extract token handling to the dedicated class
- Handling between single-user and multi-user should be performed by using inversify opportunities.